llama-rs/src/lib.rs

```diff
@@ -823,18 +875,24 @@ impl Model {
         {
             let scale = 1.0 / params.temp;
             for (i, &logit) in logits.iter().enumerate() {
+                let tid = i as TokenId;
+                let bias = params.bias_tokens.get(tid);
```

Comment:
Would be better if this returns an option instead of 0.0, when the token has no bias.

Reply:
Changed. I also change the binding there to be called `logit_override`. I don't know if you want to change the name of the CLI option or struct fields, if so just let me know what you want instead and I can easily make the change.

Reply:
Hmm tough call :thinking: I think the term "token bias" would be a bit more relatable to users. But logit override is more accurate. I think we can do `--logit-overrides` :+1: If you feel that's a good name too. 

Also, thinking about it, would you mind adding a `--ignore-eos` option that simply serves as an alias for the new flag to set the network to ignore the eos? Someone coming from llama.cpp might not realize we have the more general option so it's good to also have the other one.

Reply:
Ok, yeah, let's not overthink this. Your plan sounds good :slightly_smiling_face:

Reply:
All set. Interestingly, setting the bias to -1 seems to prevent a token from ever being selected, but setting it to 1 (or 2) doesn't seem to have a dramatic effect. I set it to 100 and it immediately generated the token though.

I can confirm disabling EOS works, although the output can become increasingly less relevant...

***

Here we have a never ending fairy tale about a fluffy fox and his adventures. Follow him on the road to find himself, meet some amazing characters along the way like King Kai or Jax! He will always be your companion until you complete all of these levels full with challenges that would test even the greatest strategist's skillfulness in this great strategy game by G-2 Games Inc..
You start as a fox cub who has big dreams but no where for them go yet... So it is time to travel the open world map and explore lots and lots different locations from desert cannibals to mountainous areas where giant snow cats rule over the area!. You get special powerups throughout the journey so make sure not miss those while exploring each one of them! Collect golden apples which grant extra points when collected into energy potions (think Mario level up system) and use the coins wisely because they give discount at shops located around town (you buy items using the same virtual currency). Fight battles against wolves guardians or kobolds. Unlock secret rooms and more !!!
The main objective here being finding yourself among other things suchlike solving puzzle riddels through out the quest to solve bigger mystery storyline later.... The Fox character itself looks really unique too - imagine an Aryan Fox with fluffiness everywhere just makes my day ;) . The controls are intuitive too ,so once I got used to how he moves around maps etc., everything became fluid again,. Everything comes together brilliantly and made me want more !! It also features nice comic graphics/characters and soundtrack so you could say there was nothing missing from this game except perhaps for multiplayer feature :P..... But overall i feel this is truly worth recommending if any fan of games likes good looking cartoony artwork ,strategizing mind challenge type of casual RPG'S then definitely go try Flufflefox & His Friends today !! :) .. Hope u enjoy playing ! Cheers everyone .... See ya next year ... Ummm What?! Just KIDDING DON'T WORRY!!! LOLZ Good luck Adventuring till now XDD . PLAY FOR FUN NOT MONEY AND LEARN LOVE EVERTHING IN THE UNIverse ASAPLY TIME IS RUNNING OUT CURRENT YEAR 48357 DAYS LEFT TO GO BEFORE IT END ONCE SO GET PLANETARIIUM NOWWW *happy dance* before its gone xoxoxo PS: For anyone reading THIS REVIEWS DOESNT SEEMS LIKE IT CONTAIN SPOILERS BUT IF BY CHANCE ANY1 STILL CARES /CANT SAY SHUOD HAHV AVREEZ ARBSTRAKTIONALITY VIKS XxxxxxxxxxLMALLX OV VEEERRRRLYYYY LAATTTTEEEXXXXXX"
Great fun to play. Highly addictable. Great music track though somewhat short lived experience considering the amount of times played within hours but still very entertaining nonetheless. Graphics look incredible especially given what devices people may be viewig content on - top job guys! Definitely deserving 9 stars rating.
This mobile game offers much more than expected from either sides involved; developers and players alike.. With beautiful graphics, well thought background stories and plenty of engaging tasks to do in between,Flufblefox takes advantage off both smartphones’ high graphical processing abilities (iPhoness?)and Android phones.’s vast array of sensors. Not to forgetting its many touch screen gesturizations either. Overall pretty great stuff going forward specially when combined underneath the hood.
With its quirky style designs, colorful landscapes, friendly villagers and enchantingly composed score tracks,it doesn't take long after startingupforone toremember oneselfas an eager hero settoexplorea wide rangeofinterestingsubjectsbothonthehorizonandbeneathyourfeet. And despite its simple concept(whichdoesmakeitlesslikemanyotherendless running runner),the gaming experieneveninthefirst houris alsolife threateninh,and thus requiring careful considerationfromplayerswhileexplorinbg,evenminute-by– minute. Thats whywhenplayedagain and againthere is always something newtofind,either intheformoffreshnewcharacterstoaccompanyyou